programs/voter-stake-registry/src/instructions/extend_stake.rs

```diff
@@ -68,6 +70,28 @@ pub fn extend_stake(
         source_mint_idx,
         VsrError::InvalidMint
     );
+    // check whether target delegate mining is the same as delegate mining from passed context
+    require_eq!(
```

Comment:
These requires are the same for 3 methods - could this check be specified on the context itself with an AccountLoader or as a method?

Reply:
I moved it to Stake as a dedicated method, just to keep them together.